### DISCUSSION TITLE
What ReCaptcha validates within workflow?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#8235] [Q&A]
We send a form as www-url-encoded using fetch. Hence we also want to validate if ReCaptcha was not bypassed.
Is there any sample of how to use ReCaptcha workflow task?

**Comment by davidhayden** [comment]
The documentation has a nice walkthrough of creating a contact form and an associated workflow that shows how to use the ReCaptcha workflow task.

https://docs.orchardcore.net/en/dev/docs/reference/modules/Forms/
